### ISSUE TITLE
Missing remote image warnings not showing up when building HTML

### ISSUE THREAD

**Issue by barbara-sfx** [sphinx-doc/sphinx#5196]
### Problem
`make clean latexpdf` or `make clean latex` gives warning messages about missing images in the step "Resolving references", but `make clean html` does not. I don't even see a Resolving References step when doing `make clean html`

#### Procedure to reproduce the problem

`make clean latex`  shows warnings on missing remote images, e.g. 
```
WARNING: Could not fetch remote image: https://raw.githubusercontent.com/.../dashboard_redshift.png [404]

```
but `make clean html` does not. So my live site now shows a number of broken images.

#### Error logs / results

https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.amazon.redshift.html

#### Expected results

Show warnings when building HTML

### Reproducible project / your project
- <link to your project, or attach zipped small project sample>

### Environment info
- OS: Mac 10.13.4
- Python version: 2.7
- Sphinx version: 1.7.5
- <Extra tools e.g.: Browser, tex or something else>

**Comment by tk0miya** [comment]
Sure. HTML builder does not fetch remote images. So the warnings are never shown.
But it would be nice if we can check its validity by linkcheck builder.

**Comment by barbara-sfx** [comment]
I have seen these messages in the past. So the warnings must have been removed in a sphinx update. If I have time, I can try to revert to an earlier version (although the last time I tried that I kind of screwed up my system due to my general unfamiliarity with some system activities). The last one I used before 1.7.5 was (I believe) 1.6.4.

-- Barbara

**Comment by tk0miya** [comment]
I guess you saw the warnings on resizing image. Actually, HTML builders emits them only on resizing.

**Comment by barbara-sfx** [comment]
What do you mean by resizing image? These graphics all take the following format:

in text:

`|image1|`

At the bottom of the file:

`.. |image1| image:: https://raw.githubusercontent.com/.../dashboard_redshift.png`

Image size is never addressed in the docs.

**Comment by tk0miya** [comment]
What I meant is giving resizing option to image directive like this:
```
.. image:: picture.png
   :width: 50%
```

AFAIK old sphinx did not emit warnings for subsutituion (cf. `|image1|`).
Please let me know if you can reproduce it.

If this is regression, I will fix this soon. If not, I will work for this after 1.8 release.

**Comment by barbara-sfx** [comment]
As my sample showed, I am not using the ..image:: directive

If time allows, I'll see if I can figure out which version gave the errors. It might have been as far back as 1.4.

**Comment by barbara-sfx** [comment]
Oh actually, they have fixed the missing images problem, so I would have to break something to test again. But I am 100% certain it worked before. I was working on a pre-release set of docs which pointed to images in a dev github repo that was still under development and did not yet contain the images the docs linked to. I just had to ignore all the warnings for a while during development. So I do think it is a regression.

**Comment by tk0miya** [comment]
Until somebody let me know reproducible information, I will keep this as "proposal", not "regression".
Thanks,

**Comment by tk0miya** [comment]
Fixed by #6022.
Thank you for proposal.
